analysis_driver/report_generation/crawler.py

```diff
@@ -15,16 +15,18 @@ def gender_alias(cls, gender):
 
     @classmethod
     def get_sample_information_from_lims(cls, sample_name):
+        lims_sample = clarity.get_sample(sample_name)
         sample_info = {
             c.ELEMENT_SAMPLE_EXTERNAL_ID: clarity.get_user_sample_name(sample_name, lenient=True),
             c.ELEMENT_SAMPLE_PLATE: clarity.get_plate_id_and_well(sample_name)[0],  # returns [plate_id, well]
             c.ELEMENT_PROVIDED_GENDER: cls.gender_alias(clarity.get_sample_gender(sample_name)),
-            c.ELEMENT_SAMPLE_SPECIES: clarity.get_species_from_sample(sample_name),
-            c.ELEMENT_SAMPLE_EXPECTED_YIELD: clarity.get_expected_yield_for_sample(sample_name)
+            c.ELEMENT_SAMPLE_SPECIES: clarity.get_species_from_sample(sample_name)
         }
-        lims_sample = clarity.get_sample(sample_name)
-        coverage = lims_sample.udf.get('Coverage')
-        if coverage:
-            sample_info[c.ELEMENT_SAMPLE_EXPECTED_COVERAGE] = coverage
+        if 'Yield for Quoted Coverage (Gb)' in lims_sample.udf:
+            sample_info[c.ELEMENT_SAMPLE_REQUIRED_YIELD] = lims_sample.udf.get('Yield for Quoted Coverage (Gb)') * 1000000000,
```

Comment:
I think these trailing commas are going to result in the crawler trying to push a tuple, e.g. `(3000000000,)`, not a number. We should fix that and add these UDFs to the mocked `clarity.get_sample` return value in `integration_tests.mocked_data` so we know they're getting pushed properly.